drivers/flash/Kconfig

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig FLASH
```

Comment:
btw. FLASH appears now twice in Kconfig. Do we need a separate symbol in NCS ?

Reply:
I do not think so, might be wrong, but this way the FLASH, in NRF drivers, gets automatically visible when you select FLASH in zephyr.